### ISSUE TITLE
question about V0.4.2

### ISSUE THREAD

**Issue by bmj12345** [pjedge/longshot#57]
Hellow sir，how can i use longshot ’V0.4.2‘? In my knowledge bioconda only relase 'V0.4.1'. when i download code, it only has ''build.rs  Cargo.toml  example_data  LICENSE  README.md '',how can i use ’longshot -bam  -ref -out‘ like this command，or just now can only use the script from v0.4.2 code.

**Comment by vibansal** [comment]
We will push the new version to bioconda soon. Meanwhile, you can compile from the source code: "cargo build --release"

**Comment by Coppini** [comment]
Tried following the instruction to compile from the source code with `cargo build --release` but it didn't work, throwing a lot of warning messages, and ending on 
```
error: build failed
``` 

Same with `cargo install --path .`:
```
error: failed to compile `longshot v0.4.2 (/home/coppini/bioinfo/wdl-nanopore/docker/longshot/longshot)`, intermediate artifacts can be found at `/home/coppini/bioinfo/wdl-nanopore/docker/longshot/longshot/target`

Caused by:
  build failed
```

I already had `longshot==0.4.1` installed with conda, on `ubuntu:20.04`, so I tried installing it on docker, on `ubuntu:18.04` as specified, but had the same problems. 

Is there any docker images or Dockerfiles available for 0.4.2?
